README.md

```diff
@@ -1 +1,12 @@
 # eq-survey-registry
+
+## Installing Couch DB
+
+1. sudo apt-get install couchdb
```

Comment:
Could we add in how to install on a mac here? It's just 'brew install couchdb' but makes it easier for future development.